libsolidity/lsp/GotoDefinition.cpp

```diff
@@ -0,0 +1,79 @@
+#include <libsolidity/lsp/GotoDefinition.h>
+#include <libsolidity/lsp/Transport.h> // for RequestError
+#include <libsolidity/lsp/Utils.h>
+#include <libsolidity/ast/AST.h>
+#include <libsolidity/ast/ASTUtils.h>
+
+#include <fmt/format.h>
+
+#include <memory>
+#include <string>
+#include <vector>
+
+using std::make_shared;
+using std::move;
+using std::string;
+using std::vector;
+
+using namespace solidity::frontend;
+using namespace solidity::langutil;
+
+namespace solidity::lsp
```

Comment:
We usually only put the declarations inside the namespace and use `using` for the definitions.